app-i18n/cldr-emoji-annotation/cldr_emoji_annotation-37.0_13.0_0_2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Emoji annotation files in CLDR"
+DESCRIPTION="This package contains the pkg-config files for development when building\
+programs that use cldr-annotations."
+HOMEPAGE="https://github.com/fujiwarat/cldr-emoji-annotation"
+COPYRIGHT="1991-2022, Unicode, Inc"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/fujiwarat/cldr-emoji-annotation/archive/${portVersion}/cldr-emoji-annotation-$portVersion.tar.gz"
+CHECKSUM_SHA256="22fbf6a96f6966edaff2bd23694568da394e064a0599ef5c6047fffe233cab16"
+SOURCE_DIR="cldr-emoji-annotation-$portVersion"
+
+ARCHITECTURES="any"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cldr_emoji_annotation$secondaryArchSuffix = $portVersion
+	devel:cldr_emoji_annotation$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku_devel$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./autogen.sh
```

Comment:
autogen.sh calls for aclocal, autoheader, autoconf and automake, it also calls configure, so maybe just drop autogen.sh here and run autoreconf -vfi ?